client/components/TextView.jsx

```diff
@@ -21,18 +21,22 @@ export default class TextView extends React.Component {
   resetText = () => {
     this.setState({
       visibleAnalytics: false,
+      value: '',
     });
   }
 
+  onChange = (e) => {
```

Comment:
I think we might want to change this method's name to "handleChange" for consistency